demo/dev/index.htm

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <script src="/checkout.js" data-paypal-checkout></script>
+    <script src="../../dist/checkout.js?debug=true" data-paypal-checkout></script>
```

Comment:
There was some confusion around `npm run dev` vs `npm run demo`. Let's change this one back since it works fine with `npm run dev`:

```suggestion
    <script src="/checkout.js" data-paypal-checkout></script>
```

Reply:
Yep.  Already on it from our conversation last week.   Thanks, @gregjopa